src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age ? <p className="Person__age">I am {person.age}</p> : null}
```

Comment:
1. Better use 

```suggestion
    {person.age && <p className="Person__age">I am {person.age}</p>}
```

2. Your code (and my suggestion) will have a problem, when `age` is `0`. Please fix it